pkg/v1/tkg/tkgconfigupdater/helper_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package tkgconfigupdater
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	sampleFilePath string
+	destDir        string
+)
+
+var _ = Describe("Tests while unzipping a file", func() {
+	Context("Validating destDir path", func() {
+		It("when file path is invalid", func() {
+			sampleFilePath = "baz"
+			destDir = "foo/../bar"
+			err := unzip(sampleFilePath, destDir)
+			Expect(err).To(HaveOccurred())
```

Comment:
nit: please make the verification more explicit (e.g checking on the message content) so we can tell the error obtain was indeed due to the fix we just added.